tests/cpp/test_mma.cpp

```diff
@@ -1410,10 +1410,14 @@ TEST_P(Blackwell1CTAM128SS, MultipleTile) {
     mma_result->axis(-2)->parallelize(ParallelType::Mma);
     mma_result->axis(-3)->parallelize(ParallelType::Mma);
     // Set the allocation domain
-    AbstractTensor s(mma_result->getLoopDomain());
     // [Mo, No, Ko, Mi, Ni, Ki] -> [Mi, |, Mo, No, Ko, Ni, Ki]
-    s.reorder({{3, 0}});
-    mma_result->setAllocationDomain(s.as<IterDomain*>(), true);
+    // Note that we intentionally reorder both the loop domain and the
```

Comment:
I just realized that in my previous PR, we were fully inlining the `MmaOp`, as a result, the TMem only has one tile, and the computed index is always zero. This is not what we want to test. So I slightly modified here to prevent the inlining of `MmaOp`.